.github/workflows/bleeding-edge.yaml

```diff
@@ -61,6 +61,7 @@ jobs:
       # are not installed by pip as specified from pyproject.toml, hence we get
       # to use the dev version of numpy at build time.
       run: |
+        python setup.py build_clib -q
```

Comment:
I think we should just keep the `pip install` line. As it stands, extensions are needlessly compiled twice (my bad, I think !).